File: Kotlin/프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.kt
class Solution {
    fun solution(s: String): String {
        var answer = ""

        var min = Int.MAX_VALUE
        var max = Int.MIN_VALUE
        for (c in s.split(" ")) {
            val cToInt = c.toInt()
            if (cToInt > max) {
                max = cToInt
            }

            if (cToInt < min) {
                min = cToInt
            }
        }

        return "$min $max"
    }
}

File: C#/프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.cs
public class Solution {
    public string solution(string s)
    {
        string[] split = s.Split(' ');

        int min = int.MaxValue;
        int max = int.MinValue;

        foreach (var s1 in split)
        {
            int i = int.Parse(s1);

            if (min > i)
            {
                min = i;
            }

            if (max < i)
            {
                max = i;
            }
        }
        
        
        return $"{min} {max}";
    }
}